sdk/core/azure-core-amqp/src/main/java/module-info.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.amqp {
+    requires com.azure.core;
+
+    requires reactor.core;
+
+    exports com.azure.core.amqp;
+    exports com.azure.core.amqp.exception;
+
+    exports com.azure.core.amqp.implementation to
+        com.azure.messaging.eventhubs;
```

Comment:
Add a FIXME or TODO in here, so that we know to revisit this (and all other 'exports .. to' lines).

Reply:
Added comments to fix these later.